File: 最长特殊序列I.cpp
class Solution {
public:
    int findLUSlength(string a, string b) {
        if(a.length() > b.length())
            return a.length();
        else if(a.length() < b.length())
            return b.length();
        else if(a == b)
            return -1;
        else
            return a.length();
    }
};

File: 最长特殊序列I.py
class Solution:
    def findLUSlength(self, a: str, b: str) -> int:
        la = len(a)
        lb = len(b)
        if la > lb:
            return la
        elif la < lb:
            return lb
        elif a == b:
            return -1
        else:
            return la
